charts/ChartConfig.tsx

```diff
@@ -696,8 +696,10 @@ export class ChartConfig {
             this.props.tab === "table" &&
             value === "chart" &&
             !this.userHasSetTimeline
-        )
+        ) {
             this.revertAutoSelectedStartYear()
+            this.props.minPopulationFilter && this.toggleMinPopulationFilter()
```

Comment:
It doesn't feel right to call toggleMinPopulationFilter (a user command/action) in a setter for something else.

Unless I'm misreading this it says "if the pop filter is on, turn it off, and update the URL". Is that what we want?

Reply:
Currently, the "hide countries" checkbox is only available on scatterplots on the Chart tab. This PR causes the checkbox to be visible across _all_ tables, not just Scatterplot tables. So a user might hide small countries on the Table, return to Chart, and no longer have a checkbox to uncheck and see small countries again.

I agree it can be improved. One way to is to remember the option state on Chart and restore it rather than just disable the filter. This way, if the option was already enabled, it will stay enabled on switching back.

Re. using the toggle method -- for some reason I had thought `minPopulationFilter` was marked private, but looking back I see it isn't, so I can just manually change that as well.

Reply:
Gotcha, thanks. Maybe we should just show the hideCountries toggle on all charts then (keeping in place the condition that we check to see if there are countries that would be hidden first)? Whatever you think is best.

Ideally we can get a good transform architecture in soon and make our filters simpler.

Reply:
Since non-scatters rarely show many entities and we're starting to run into places where we actually have too many controls in the footer, I'm actually in favor of hiding this control unless necessary.

Reply:
Pushing some temporary imperative code to remember the state. Would like to move to transform-specific filter columns ASAP, as I messaged you about.